samples/change-notification/csharp/README.md

```diff
@@ -0,0 +1,89 @@
+﻿### Change Notification sample
+
+Bot Framework v4 ChangeNotification sample.
+
+This sample app demonstrates sending change notifications to user presence in Teams based on user presence status.
+
+
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+  A) Or from Visual Studio code
+
+  - Launch Visual Studio code
+  - File -> Open Folder
+  - Navigate to `samples/graph-change-notification/csharp` folder
+  - Select `ChangeNotification` folder
+  - Press `F5` to run the project
+
+  B) Run ngrok - point to port 3978
+
+   ```bash
+     ngrok http -host-header=rewrite 3978
+   ```  
+    
+## Instruction on setting connection string for bot authentication on the behalf of user
+1. In the Azure portal, select your resource group from the dashboard.
+
+2. Select your bot channel registration link.
+
+3. Open the resource page and select Configuration under Settings.
+
+4. Select Add OAuth Connection Settings.
+![image](https://user-images.githubusercontent.com/85864414/121879805-df15cb00-cd2a-11eb-8076-1236ccb1bbfc.PNG)
+5. Complete the form as follows:
+
+![image](https://user-images.githubusercontent.com/85864414/122000240-1d16fb80-cdcc-11eb-8aeb-a1dc898f947e.PNG)
+
+a. Enter a name for the connection. You'll use this name in your bot in the appsettings.json file. For example BotTeamsAuthADv1.
+
+b. Service Provider. Select Azure Active Directory. Once you select this, the Azure AD-specific fields will be displayed.
+
+c. Client id. Enter the Application (client) ID that you recorded for your Azure identity provider app in the steps above.
+
+d. Client secret. Enter the secret that you recorded for your Azure identity provider app in the steps above.
+
+e. Grant Type. Enter authorization_code.
+
+f. Login URL. Enter https://login.microsoftonline.com.
+
+g. Tenant ID, enter the Directory (tenant) ID that you recorded earlier for your Azure identity app or common depending on the supported account type selected when you created the identity provider app.
+h. For Resource URL, enter https://graph.microsoft.com/
+i. Provide  Scopes like "Presence.Read, Presence.Read.All"
+![image](https://user-images.githubusercontent.com/85864414/121880473-af1af780-cd2b-11eb-8166-837425ef186f.PNG)
+
+## Instruction for appsetting
+1. Provide appId and appPassword in the appsetting that is created in Azure.
+2. Provide UserId in appsetting for which you want to subscribe the user presence.
+3. Provide the ngrok url as  "BaseUrl" in appsetting on which application is running on.
+
+
+## Concepts introduced in this sample
+# After sucessfully installation of app you will get a sign in button. When sign in is complete then you get your current status in adapative card
```

Comment:
Could you please correct the formatting of these instructions? Currently it's showing with heading formattng. 
![image](https://user-images.githubusercontent.com/31851992/122673625-df144000-d1ee-11eb-91d6-b5e48afac836.png)